cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -223,10 +223,10 @@ public synchronized CryptoVault load(final Session<?> session, final PasswordCal
         return this.unlock(session, prompt, bookmark, passphrase);
     }
 
-    protected VaultConfig readVaultConfig(final Session<?> session) throws BackgroundException {
+    protected VaultConfig parseVaultConfigFromJWT(final Session<?> session) throws BackgroundException {
```

Comment:
I proposed this one should be `readVaultConfigFromJWT´ to reflect the fact that it is reading from remote using the `session` parameter.